clang/test/CodeGenSYCL/inline_asm.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsycl-enable-optimizations -triple spir64-unknown-unknown-sycldevice -emit-llvm -x c++ %s -o - | FileCheck %s
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -triple spir64-unknown-unknown-sycldevice -emit-llvm -x c++ %s -o - | FileCheck %s
```

Comment:
Should this line have -fno-sycl-std-optimizations?

Reply:
Original test is compiled with enabled optimizations, so I kept it that way.